main.py

```diff
@@ -0,0 +1,77 @@
+import os
+import json
+import requests
+import urllib.request
+from datetime import datetime
+
+from dotenv import load_dotenv
+from typing import Optional, Dict, List
+from fastapi import FastAPI, Response, Query
+
+import functions
+
+load_dotenv()
+app = FastAPI()
+
+
+@app.get("/comics/current")
+def getCurrentComic(response: Response):
+    try:
+        comicData = functions.loadJSON('https://xkcd.com/info.0.json')
+    except urllib.error.HTTPError as error:
+        response.status_code = error.code
+        return error
+    return functions.processComicData(comicData)
+
+
+@app.get("/comics/many")
+def getManyComics(response: Response, comic_ids: List[int] = Query(...)):
+    output = []
+    history = []
+    for comicId in comic_ids:
+        if comicId in history:
```

Comment:
this looks simple and nice, BUT there's one problem with it - when `history` is a list, what _really_ happens is: the interpreter needs to iterate through all elements until it finds `comicId`.

If `history` was a set or a list, then this operation would be much cheaper. 

Look at this example:

```
$ python -m timeit -u usec -n 10000 -s "a = list(range(100))" "99 in a"
10000 loops, best of 5: 1.13 usec per loop
 $ python -m timeit -u usec -n 10000 -s "a = set(range(100))" "99 in a"
10000 loops, best of 5: 0.0278 usec per loop
```